code/game/objects/items/devices/PDA/PDA.dm

```diff
@@ -286,6 +286,12 @@ GLOBAL_LIST_EMPTY(PDAs)
 						dat += "<li><a href='byond://?src=[REF(src)];choice=Toggle Door'>[PDAIMG(rdoor)]Toggle Remote Door</a></li>"
 					if (cartridge.access & CART_DRONEPHONE)
 						dat += "<li><a href='byond://?src=[REF(src)];choice=Drone Phone'>[PDAIMG(dronephone)]Drone Phone</a></li>"
+					if (cartridge.access & (CART_MANIFEST | CART_STATUS_DISPLAY))
```

Comment:
Id rather have this on status_display instead of manifest in case we make the manifest public in the future
```suggestion
					if (cartridge.access &CART_STATUS_DISPLAY)
```

Reply:
Alex, wouldn't (CART_MANIFEST | CART_STATUS_DISPLAY) be CART_MANIFEST and CART_STATUS_DISPLAY?

Reply:
You see, bitflags are fucky. This works as intended if the intention was the make it trigger on either CART_MANIFEST or CART_STATUS_DISPLAY